conf.pri.in

```diff
@@ -12,7 +12,7 @@ QMAKE_CXXFLAGS += @QBT_CONF_EXTRA_CFLAGS@
 EXTERNAL_INCLUDES = @QBT_CONF_INCLUDES@
 EXTERNAL_INCLUDES -= $$QMAKE_DEFAULT_INCDIRS
 # added /usr/local/include due to Qt 5.7.0 bug on macOS
-macx: EXTERNAL_INCLUDES += "/usr/local/include"
+macos: EXTERNAL_INCLUDES += "/usr/local/include"
```

Comment:
why change it? The `macx` value is still valid.

Reply:
The `macx` is an older define which pollutes us with unnecessary guarding, like the [one](https://github.com/qbittorrent/qBittorrent/pull/16037#discussion_r782636088) you saw above. `macos` has a cleaner define.

It is hard for me to track what qmake includes, so here is the test build with qt5:
branch: https://github.com/Chocobo1/qBittorrent/commits/28a2967fa54ee4a5760e4a639a266ccfcb9d7d2a
CI log: https://github.com/Chocobo1/qBittorrent/runs/4778236341?check_suite_focus=true#step:7:16

You can see that it still build correctly and the dbus is correctly handled (i.e. not included).